lib/anyway/config.rb

```diff
@@ -43,22 +43,22 @@ def config_name(val = nil)
       #
       #   my_config = Anyway::Config.for(:my_app)
       #   # will load data from config/my_app.yml, secrets.my_app, ENV["MY_APP_*"]
-      def for(name)
-        new(name, false).load_from_sources
+      def for(explicit_values: {}, name: nil)
+        new(explicit_values: explicit_values, name: name, load: false).load_from_sources
       end
     end
 
     attr_reader :config_name
 
-    def initialize(config_name = nil, do_load = true)
-      @config_name = config_name || self.class.config_name
+    def initialize(explicit_values: {}, name: nil, load: true)
```

Comment:
Looks that I've proposed a wrong solution: there gonna be a breaking change.

Let's do the following:

```ruby
def initialize(config_name = nil, do_load = nil, name: nil, load: true, overrides: {})
  unless config_name.nil? && do_load.nil?
    warn "[Deprecated] Positional arguments for Anyway::Config#initialize will be removed in 1.2.0. Use keyword arguments instead: initialize(name:, load:, overrides:)"
  end

  name = config_name unless config_name.nil?
  load = do_load unless do_load.nil?

  ...
end
```

And let's rename `explicit_values` to `overrides`.